docs/macios/xcsync.md

```diff
@@ -35,13 +35,13 @@ The tool has two commands: `generate` and `sync`. Use `generate` to create an Xc
 Install .NET xcsync globally with the following command:
 
 ```dotnetcli
-dotnet tool install -g xcsync
+dotnet tool install -g dotnet-xcsync
 ```
 
 ## Usage
 
-1. The [.NET MAUI extension](https://aka.ms/mauidevkit-marketplace) provides support for xcsync. Ensure you have the latest version.
-2. Open the command palette (Cmd + Shift + P) and search for the `.NET MAUI: xcsync- Generate an Xcode project` and `.NET MAUI: xcsync- Sync from an Xcode project` commands.
+1. The [.NET MAUI extension](https://aka.ms/mauidevkit-marketplace) provides support for xcsync. Ensure you have the latest version on macOS.
```

Comment:
I think the fact that this tool only works on macOS needs to be more explicit. As in "The xcsync commands in the .NET MAUI extension are only available when using VS Code on a Mac."